specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -5313,6 +5314,15 @@
       },
       "description": "Specifies the storage settings for the virtual machine disks."
     },
+    "AdditionalCapabilities": {
+        "properties": {
+            "ultraSSDEnabled": {
```

Comment:
This does not follow ARM guideline. Even if Gaurav signed-of, I checked with @veronicagg that you should get a suppression rule approved by ARM:
https://github.com/Azure/adx-documentation-pr/wiki/Swagger-Validation-Errors-Suppression

Reply:
@rohbat